exporter/awscloudwatchlogsexporter/go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/google/uuid v1.5.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil v0.92.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs v0.92.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.0.0-00010101000000-000000000000
```

Comment:
Wrong version?

Reply:
it's replaced, will get fixed next release. I can fix it now.

Reply:
```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.92.0
```